SPMZipArchive.xcodeproj/xcshareddata/xcschemes/ziparc.xcscheme

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
+   LastUpgradeVersion = "9999"
```

Comment:
That's a high version ;)

Reply:
Umm, what is that mean? Sorry 😄

Reply:
A normal LastUpgradeVersion for Xcode 11 is `"1100"`. Example from ZipArchive itself with Xcode 10.2:
https://github.com/ZipArchive/ZipArchive/blob/3078747d91537e8c5cea89971bb21e7a1d687d0c/ZipArchive.xcodeproj/xcshareddata/xcschemes/ZipArchive-iOS.xcscheme#L3

Reply:
I think, this is because the xcodeproj is generated by SwiftPackageManager.

Reply:
OK, maybe a bug with the Beta. Note that my main question regarding the PR is on https://github.com/ZipArchive/ZipArchive/pull/523#issuecomment-506274918